packages/xo-web/src/xo-app/index.js

```diff
@@ -76,11 +76,15 @@ const BODY_STYLE = {
 @routes('home', {
   about: About,
   backup: Backup,
-  'backup-ng': BackupNg,
+  'backup-ng/*': {
+    onEnter: ({ location }, replace) =>
+      replace(location.pathname.replace('/backup-ng', '/backup')),
```

Comment:
Did backup-ng have the exact same sub-routes as backup?

Reply:
No, there are sub-routes for backup-ng only(ex:  `new\metadata`, `\health` ...).

Reply:
Ok but it doesn't matter anyway, all the routes that *were* on `/backup-ng` *will be* on `/backup` for sure since we're moving everything, my bad.